app/src/main/java/org/mozilla/fenix/logins/SavedLoginsAdapter.kt

```diff
@@ -14,11 +15,12 @@ private sealed class AdapterItem {
 }
 
 class SavedLoginsAdapter(
+    private val context: Context,
     private val interactor: SavedLoginsInteractor
 ) : ListAdapter<SavedLoginsItem, SavedLoginsListItemViewHolder>(DiffCallback) {
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): SavedLoginsListItemViewHolder {
-        val view = LayoutInflater.from(parent.context).inflate(viewType, parent, false)
+        val view = LayoutInflater.from(context).inflate(viewType, parent, false)
```

Comment:
I'm pretty sure we need to inflate the correct view here. I'm going to close this and we'll open a new PR that fixes this